taipy/gui/utils/types.py

```diff
@@ -197,6 +197,23 @@ def get(self):
     def get_hash():
         return _TaipyBase._HOLDER_PREFIX + "Di"
 
+class _TaipyTime(_TaipyBase):
+    def get(self):
+        val = super().get()
+        if isinstance(val, time):
+            val = _date_to_string(val)
+        elif val is not None:
+            val = str(val)
+        return val
+
+    def cast_value(self, value: t.Any):
+        if isinstance(value, str):
+            return _string_to_date(value)
```

Comment:
I'm not sure _string_to_date returns a python time instance ?

Reply:
It was not, but now it's